.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```
@@ -401,6 +401,12 @@ jobs:
           run: python -m scripts.build --prod_env
         - name: Run Desktop Acceptance Test ${{ matrix.suite }}
           run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite }} --prod_env
+        - name: Uploading generated test to angular modules as an artifact
```

From the description, I think people aren't going to understand what these are. Maybe this?

```suggestion
        - name: Uploading generated test to angular module mappings as an artifact
```

---

Done, though changed to mapping since its a single mapping for one test in this case.